queries.sql

```diff
@@ -23,3 +23,57 @@ select * from animals where animal_name <> 'Gabumon';
 
 /*All animals with a weight between 10.4kg and 17.3kg*/
 select * from animals where weight_kg between 10.4 and 17.3;
+
+
+/* Transactions 1: Set species to 'unspecified' and Rollback*/
+begin;
+update animals set species = 'unspecified';
+
+rollback;
```

Comment:
Great job so far. It would be great to verify that the changes were made in the table before and after `rollback` the transaction. You just need to add `SELECT species from animals` before and after `rollback`. So that you can verify whether the change was made or not and is undone. Kindly, add it.

![image](https://github.com/iankiema/Vet-Clinic-Database/assets/101523582/73835db3-8592-4a15-87ac-f7b24293ec59)